x/gamm/keeper/pool_service_test.go

```diff
@@ -41,93 +41,53 @@ var (
 func (suite *KeeperTestSuite) TestCreateBalancerPool() {
 	params := suite.App.GAMMKeeper.GetParams(suite.Ctx)
 
+	// get raw pool creation fee(s) as DecCoins
 	poolCreationFeeDecCoins := sdk.DecCoins{}
 	for _, coin := range params.PoolCreationFee {
 		poolCreationFeeDecCoins = poolCreationFeeDecCoins.Add(sdk.NewDecCoin(coin.Denom, coin.Amount))
 	}
 
-	func() {
-		keeper := suite.App.GAMMKeeper
+	// we keep this account empty to test attempts with insufficient balance
+	emptySender := suite.TestAccs[1]
 
-		// Try to create pool without balances.
-		msg := balancer.NewMsgCreateBalancerPool(suite.TestAccs[0], defaultPoolParams, defaultPoolAssets, defaultFutureGovernor)
-		_, err := keeper.CreatePool(suite.Ctx, msg)
-		suite.Require().Error(err)
-	}()
-
-	// TODO: Refactor this to be more sensible.
-	// The struct should contain a MsgCreateBalancerPool.
-	// then the scaffolding should test pool creation, and check if it was a success or not.
-	// (And should be moved to balancer package)
-	// PoolCreationFee tests should get their own isolated test in this package.
+	// TODO: should be moved to balancer package
```

Comment:
@ValarDragon what did you mean by this comment?

Reply:
That it should be moved to the balancer package?

(Seeing as its a create Balancer pool message =p )